### DISCUSSION TITLE
Can carousel play automatically?

### DISCUSSION THREAD

**Discussion by hutong9** [saadeghi/daisyui#1587] [Ideas / feature requests]
Can carousel play automatically?

**Comment by saadeghi** [comment]
daisyUI's carousel is just a simple CSS-only solution. It is possible to make a autoplay carousel using CSS but it would be a hacky and dirty code.  

You can write a JS code to check the radio buttons with a timer and it will play automatically  
but I would suggest to use a JS plugin for carousel if you need more functionality. Because JS plugins can change DOM nodes on demand and it allows them to provide more flexibility and more functionality.
